src/mentors.js

```diff
@@ -0,0 +1,17 @@
+import { mentors } from "./mentors.json";
+
+const waves = 2;
+
+// Construct a 2-dimensional array where `mentorIds[0]` is the list of mentors
+// in the first wave.
+// The mentor IDs should eventually be converted to unique IDs instead of array
+// indices.
+const mentorIds = Array.from({ length: waves }, (_, waveIndex) =>
+  mentors
+    // Construct a new object to preserve the index before filtering.
+    .map((mentor, index) => ({ mentor, index }))
+    .filter(({ mentor }) => mentor.wave === waveIndex + 1)
```

Comment:
Could we stick to 0-based `waveIndex` to avoid confusion in the future?

Reply:
I think given all the fields in the json, maybe not? Maybe a few years into the future, when we have a million waves, it might be more intuition friendly to stick to 1-based? Esp if we wanna look up a mentor and see what wave they belong to (just for reference).

Reply:
@RyanRTJJ Preferably the `waveIndex` would be generated by the Python script so it should be used only internally in code. As I proposed in https://github.com/AdvisorySG/mentorship-page/pull/87#discussion_r418957946 there'll be the ability to label the waves as we wish, so I think it shouldn't be a big issue to stick with 0-indexing :)